README.md

```
@@ -1,175 +1,110 @@
+
+![Web Interface Screenshot](./media/Webinterface.png)
+
+### Key Options:
+
+- **LSL Streaming**: Choose a protocol (`Wi-Fi`, `Bluetooth`, `Serial`).  
+- **CSV Logging**: Data saved as `ChordsPy_{timestamp}.csv`.  
+- **Applications**: Multiple Applications can be Launch from the Interface simultaneously(e.g., `EEG Tug of War`). 
+
+## Connection Guide  
+
+#### WIFI Connection  
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)